src/knx/config.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#ifndef NO_KNX_CONFIG
```

Comment:
What is this for? I think this got mixed in?

Reply:
Right
This was to avoid to modify files inside library tree and keep all externals always vanilla.
Specific configurations should better be placed in settings defines.
I noticed a similar behaviour where platforms are linked with specific media.